fulvio-almeida/1.ii/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+
+<body>
+    <h1>Fulvio Almeida Silva</h1>
+    <p>Brasileiro, solteiro, 25 anos.</p>
+
+    <p>Rua Américo Brasiliense, 2201 - Apto 124</p>
+    <p>Chacára Santo Antônio - São Paulo - SP</p>
+    <p>Telefone: 35 99808 3497</p>
+    <p>E-mail: fulvioalmeidasilva@gmail.com</p>
+    <p>LinkedIn: https://br.linkedin.com/pub/fulvio-almeida-silva/2b/a01/b08</p>
+
+    <h2>Qualificações</h2>
+    <ul>
+        <li>Experiência nas tecnologias JAVA, JAVA EE, VBA, WEB (PHP, HTML, CSS, HTML5, Javascript, JSP, Struts, Angular.JS e Bootstrap);</li>
+        <li>Experiência em PostgreSQL, MySQL, SQL SERVER, Oracle, Firebird;</li>
+        <li>Experiência com liderança e grande facilidade para trabalhar em equipe;</li>
+        <li>Atuação visando sempre melhor custo benefício, otimizando custos, atendendo a prazos e qualidade;</li>
+        <li>Programador com habilidades comerciais e de negociação;</li>
+        <li>Conhecimento intermediários em: Android, Windows Phone;</li>
+        <li>Inglês intermediário;</li>
+        <li>Suporte in loco em clientes.</li>
+    </ul>
+    <h3>Formação Acadêmica</h3>
+    <ul>
+        <li>Pós-graduação Desenvolvimento de Aplicações para Dispositivos Móveis e Cloud Computing – INATEL – Instituto Nacional de Telecomunicações – <a href="http://www.inatel.br" target="_blank">www.inatel.br</a> – 2017/2019</li>
+        <li>Graduação: Engenharia de Computação – INATEL - Instituto Nacional de Telecomunicações – <a href="http://www.inatel.br" target="_blank">www.inatel.br</a> - 2010/2014.</li>
+    </ul>
+
+    <h4>Experiência Profissional</h4>
```

Comment:
Is it a sub-title of your graduation? 
Ref.: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements